pkgs/development/python-modules/intbitset/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, six
+, nose
+}:
+buildPythonPackage rec {
+  pname = "intbitset";
+  version = "2.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "44bca80b8cc702d5a56f0686f2bb5e028ab4d0c2c1761941589d46b7fa2c701c";
+  };
+
+  patches = [
+    # fixes compilation on aarch64 and determinism (uses -march=core2 and
+    # -mtune=native)
+    ./remove-impure-tuning.diff
```

Comment:
```suggestion
    ./remove-impure-tuning.patch
```

Reply:
They both work. This isn't actually a patch, but generated by `git diff > remove-impure-tuning.diff`.

However, I can make it a real patch, if you think that's necessary.

Reply:
No need to make it a real patch, but I think the convention still is to give it the extension `.patch`.

Reply:
imho calling it .patch is misleading if it contains a diff. Since a patch is the convention, and the list is called patches, let's make it a patch called .patch.